src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -138,9 +138,11 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <SourcesWithoutContent Include="@(_TestCopyLocalByFileNameWithoutDuplicates)" Exclude="@(ContentWithTargetPath->'%(FileName)%(Extension)')" />
-      <SourcesToCopyToTestDir Include="@(SourcesWithoutContent->'%(SourcePath)');@(ContentWithTargetPath)" />
-      <DestinationsForTestDir Include="@(SourcesWithoutContent->'$(TestPath)$(TestTargetFrameworkFolder)\%(Filename)%(Extension)');@(ContentWithTargetPath->'$(TestPath)$(TestTargetFrameworkFolder)\%(TargetPath)')" />
+      <SourcesToCopyToTestDir Include="@(RunTestsForProjectInputs)" />
```

Comment:
If we are going to copy these in msbuild then we should also copy them into in the helix runner script as well, or we should always do this copying in the runtest scripts.